packages/lib/shared/services/api/create-lbp.graphql

```diff
@@ -0,0 +1,3 @@
+mutation CreateLBP($input: CreateLBPInput!) {
```

Comment:
is this not generated automagically like the other api queries?

Reply:
wait are all the `.graphql` files generated auto-magically??!

i had no idea. i assumed you pick and choose which fields you want to take or omit